config/config.go

```diff
@@ -431,6 +431,29 @@ type HealthConfig struct {
 	Rate []Rate `yaml:"rate,omitempty" json:"rate,omitempty"`
 }
 
+// VersionRange version ranges
+type VersionRange struct {
+	Length      int
+	MeshVersion []string
+	KialiLow    []string
+	KialiHigh   []string
```

Comment:
who/what/where defines these values?

Reply:
According to https://kiali.io/docs/installation/installation-guide/prerequisites/#istio-version-compatibility

Reply:
Yes, but where in the code or config is it set? This is something we are going to have to maintain "somewhere" - where is that matrix coded (i.e. where are the json values coming from that fill in this "VersionRange"?)

Reply:
Thanks @jmazzitelli, I have added a yaml to record versions.